src/nbl/video/COpenGLESFunctionTable.h

```diff
@@ -13,7 +13,10 @@ namespace video
 /**
 Extensions being loaded:
```

Comment:
you mean required extensions?

If so then `multi_draw_indirect` cannot be on that list

Reply:
no, all extensions being loaded